### PR TITLE
A few small improvements to the tests.

### PR DESCRIPTION
**Pull request by yfarjoun**
- made index created during test get deleted in the end
- tightened TheoreticalHetSensitivity test intervals since it has been made deterministic.

### CODE REVIEW

**Comment by yfarjoun** [comment]
@ktibbett for you to review or assign.

File: src/tests/java/picard/analysis/TheoreticalSensitivityTest.java [hunk 1]
```diff
@@ -202,16 +202,16 @@ public void testHetSensDistributions() throws Exception {
         final File wgsSampledMetricsFile = new File(TEST_DIR, "test_Solexa-316269_sampled.wgs_metrics");
 
         return new Object[][] {
-                {.9130, wgsMetricsFile},
-                {.9784, hsMetricsFile},
-                {.9562, targetedMetricsFile},
-                {.9892, wgsSampledMetricsFile}
+                {0.897_342_54, wgsMetricsFile},
```
Comment (bradtaylor) [inline src/tests/java/picard/analysis/TheoreticalSensitivityTest.java]: These magics are pretty specific. How are they determined? Perhaps a comment?

**Comment by bradtaylor** [comment]
Asked for a comment on the new values. Otherwise 👍

File: src/tests/java/picard/analysis/TheoreticalSensitivityTest.java (hunk 1 above)
Comment (yfarjoun) [inline src/tests/java/picard/analysis/TheoreticalSensitivityTest.java]: do you want a comment in the code? I'm treating these as regression tests. the original "approximate" values were calculated independently in the original R implementation.

**Comment by yfarjoun** [comment]
added a line explaining (kindof) where the numbers came from

**Comment by yfarjoun** [comment]
closing and opening to check if it still passes tests..

**Comment by yfarjoun** [comment]
I noticed that I had been given an 👍 and didn't act on it yet...

**Pull request merged**